include/sir/config.h

```diff
@@ -123,7 +123,7 @@
  *   15:13:41 Fri 9 Jun 23 (-0600)
```

Comment:
Comment is out of sync. I don't really think it's necessary to change the format either; people generally know it's not 1923 or 2023, and that their time zone is -600 or whatever.